demo/kserve/deploy-remove.md

```diff
@@ -1,144 +1,150 @@
-~~~
-
-You can run the following grpcurl command for all tokens in a single call:
-~~~
-export KSVC_HOSTNAME=$(oc get ksvc caikit-example-isvc-predictor -n ${TEST_NS} -o jsonpath='{.status.url}' | cut -d'/' -f3)
-grpcurl -insecure -d '{"text": "At what temperature does liquid Nitrogen boil?"}' -H "mm-model-id: flan-t5-small-caikit" ${KSVC_HOSTNAME}:443 caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict
-~~~
-The expected response should be similar to:
-~~~
-{
-  "generated_token_count": "5",
-  "text": "74 degrees F",
-  "stop_reason": "EOS_TOKEN",
-  "producer_id": {
-    "name": "Text Generation",
-    "version": "0.1.0"
-  }
-}
-~~~
-
-### Streams of token
- You can run the following grpcurl command for streams of token:
-~~~
-grpcurl -insecure -d '{"text": "At what temperature does liquid Nitrogen boil?"}' -H "mm-model-id: flan-t5-small-caikit" ${KSVC_HOSTNAME}:443 caikit.runtime.Nlp.NlpService/ServerStreamingTextGenerationTaskPredict
-~~~
-The expected response should be similar to:
-~~~
-{
-  "details": {
-    
-  }
-}
-{
-  "tokens": [
-    {
-      "text": "▁",
-      "logprob": -1.599083423614502
-    }
-  ],
-  "details": {
-    "generated_tokens": 1
-  }
-}
-{
-  "generated_text": "74",
-  "tokens": [
-    {
-      "text": "74",
-      "logprob": -3.3622500896453857
-    }
-  ],
-  "details": {
-    "generated_tokens": 2
-  }
-}
-....
-~~~
-
-
-## Remove/undeploy LLM model
-
-To remove/undeploy a model, you can simply delete the Inference Service:
-
-~~~
-oc delete isvc --all -n ${TEST_NS} --force --grace-period=0
-~~~
-If necessary, delete the Minio resources by deleting the Minio namespace:
-~~~
-oc delete ns ${TEST_NS} ${MINIO_NS}
-~~~
-
-# Script-based model deployment and removal
-
-## Deploy a sample LLM model
-
-~~~
-./scripts/test/deploy-model.sh
-~~~
-
-## Perform inference with a gRPC call
-~~~
-./scripts/test/grpc-call.sh
-~~~
-
-## Delete a sample model and Minio
-~~~
-./scripts/test/delete-model.sh
\ No newline at end of file
+# Deploying an LLM model with the Caikit+TGIS Serving runtime
+
+There are two options for deploying and removing an LLM model:
+
+* By following the step-by-step commands that are described in this document.
+
+* By running short scripts as described in [Using scripts to deploy an LLM model with the Caikit+TGIS Serving runtime](deploy-remove-scripts.md).
+
+In this procedure, you deploy an example Large Learning Model (LLM) model, [flan-t5-small](https://huggingface.co/google/flan-t5-small), with the Caikit+TGIS Serving runtime. 
+
+Note: The **flan-t5-small** LLM model has been containerized into an S3 MinIO bucket. For information on how to containerize your own LLM model into a MinIO bucket for testing, see [Create your own MinIO image](/demo/kserve/create-minio.md).
+
+**Prerequisites**
+
+* You installed the **Caikit-tgis-serving** image as described in the [Caikit-TGIS-Serving README file](/docs/README.md).
```

Comment:
Hmm it's not instaling image exactly. Instead I think we should say smth along the lines of "You installed the pre-requisites and KServe/Caikit/TGIS stack as described in the installation instructions"